changed_from_main.py

```diff
@@ -0,0 +1,89 @@
+"""
+This script determines whether one of the projects in the repo or any of its
+dependencies are different from the version on the `main` branch.
+
+This is useful for running tests only on projects that have changed.
+
+This script should depend only on Python 3.6+ standard libraries.
+"""
+import argparse
+import re
+import os
+import subprocess
+from typing import Dict, Sequence
+import sys
```

Comment:
sys seems to be unused

Reply:
Fixed. I've gotten to a point of assuming all my imports are used because pre-commit will catch it, and forgot pre-commit isn't set up yet.